widget/editor.go

```diff
@@ -670,14 +882,15 @@ func (e *Editor) prepend(s string) {
 	if e.SingleLine {
 		s = strings.ReplaceAll(s, "\n", " ")
 	}
+	e.Delete(0) // Delete any selection first. Might move the caret.
```

Comment:
Why can e.Delete(0) "move the caret"?

Reply:
Good catch.  That comment descends from when PgUp/Down could move the caret without dismissing the selection, which they can't any more, so I'll remove it.